cql3/statements/modification_statement.cc

```diff
@@ -45,6 +45,7 @@
 #include "cql3/restrictions/single_column_restriction.hh"
 #include "validation.hh"
 #include "db/consistency_level_validations.hh"
+#include "db/system_keyspace.hh"  // Decl. of `is_system_keyspace()'
```

Comment:
If anyone uses something else from this header in the future, I don't think they'll remember to update this comment.